zcutil/release-notes.py

```diff
@@ -20,14 +20,18 @@
 ]
 
 author_aliases = {
-    'Simon': 'Simon Liu',
-    'bitcartel': 'Simon Liu',
-    'EthanHeilman': 'Ethan Heilman',
     'Ariel': 'Ariel Gabizon',
```

Comment:
This seems to be a dupe with below.. is that deliberate?

Reply:
Yes.

Reply:
Yes this is correct `Ariel`->`Ariel Gabizon` and `arielgabizon`->`Ariel Gabizon`. He has multiple aliases.